assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -65,32 +105,49 @@ export const actions = {
 		};
 	},
 
-	receiveAccountsPropertiesProfilesCompleted() {
-		return {
-			payload: {},
-			type: RECEIVE_ACCOUNTS_PROPERTIES_PROFILES_COMPLETED,
-		};
+	*resetAccounts() {
+		const registry = yield Data.commonActions.getRegistry();
+
+		yield { type: RESET_ACCOUNTS };
+
+		return registry.stores[ STORE_NAME ].getActions()
+			.invalidateResolutionForStoreSelector( 'getAccounts' );
 	},
 
-	receiveAccountsPropertiesProfilesFailed( error ) {
-		invariant( error, 'error is required.' );
+	*selectAccount( accountID ) {
+		invariant( isValidAccountSelection( accountID ), 'A valid accountID is required to select.' );
 
-		return {
-			payload: { error },
-			type: RECEIVE_ACCOUNTS_PROPERTIES_PROFILES_FAILED,
-		};
+		const registry = yield Data.commonActions.getRegistry();
+		registry.dispatch( STORE_NAME ).setAccountID( accountID );
+		registry.dispatch( STORE_NAME ).setPropertyID( '' );
+		registry.dispatch( STORE_NAME ).setInternalWebPropertyID( '' );
+		registry.dispatch( STORE_NAME ).setProfileID( '' );
+
+		if ( ACCOUNT_CREATE === accountID ) {
+			return;
```

Comment:
I suppose we could... 🤔 
It would set the `_CREATE` options for cascading values and clear the selected internal web property ID (if any). I'm hesitant to make this change since it isn't used at all. Also, creating a property and profile are part of the provisioning process, so I would be concerned about potentially duplicating those actions. Let's hold off on this for now and add it when it's needed for the flow, along with tests, etc.